### PR TITLE
For #217: Tests for assigning implementation (history)

### PR DESCRIPTION
**Pull request by paulodamaso**
For #217: Tests for assigning implementation (history)
- Added test to check if `Item` employee changes on assign
- Created `Item.item_history()` method signature to return `ItemHistory` collection from `Item`
- Added test to check if `Item` history have `ItemHistory` added on assign
- Left puzzle for `Item.item_history()` implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#263](https://github.com/timelesslounge/timelessis/issues/263) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#263](https://github.com/timelesslounge/timelessis/issues/263) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/ecc7d441-00cd-47ea-abf8-3e5f42b7ac6f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Job `gh:timelesslounge/timelessis#263` already assigned to @vryazanov, can't assign to @vryazanov

File: tests/test_items.py [hunk 1]
```diff
@@ -27,10 +28,50 @@ def test_new_item():
...
+    second_employee_id=60
+    new_item = Item(
+        id=1,
+        name="Duck Eggs",
+        stock_date=datetime.utcnow,
+        comment="Eggs from ducks",
+        company_id=223,
+        employee_id=first_employee_id
+    )
+    new_item.assign(emp_id=second_employee_id)
```
Comment (vryazanov) [inline tests/test_items.py]: @paulodamaso  Signature of `assign` is `assign(self, employee):`, so you should pass an instance of `Employee`, not  `id`

File: tests/test_items.py [hunk 2]
```diff
@@ -27,10 +28,50 @@ def test_new_item():
...
+    new_item = Item(
+        id=1,
+        name="Nanomedikit",
+        stock_date=datetime.utcnow,
+        comment="Medikit with nano particles",
+        company_id=223
+    )
+    employee_id=15
+    assert not new_item.employee_id
+    new_item.assign(emp_id=employee_id)
```
Comment (vryazanov) [inline tests/test_items.py]: @paulodamaso the same

**Comment by paulodamaso** [comment]
@vryazanov Done, please take a look

**Comment by vryazanov** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@paulodamaso check rultor and codeclimate

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vryazanov[/z](https://www.0crat.com/u/vryazanov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#263](https://github.com/timelesslounge/timelessis/issues/263) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vryazanov According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vryazanov** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@vryazanov thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Pull request merged**